.env

```diff
@@ -0,0 +1 @@
+VUE_APP_DEFAULT_PORT=8001
```

Comment:
We don't usually add `.env` to git, add `.env.example` instead. And ignore this one.